x-pack/plugins/apm/server/routes/transactions.ts

```diff
@@ -26,9 +26,13 @@ export function initTransactionsApi(server: Server) {
       const { transactionId } = req.params;
       const { traceId } = req.query as { traceId: string };
       const setup = setupRequest(req);
-      const transaction = await getTransaction(transactionId, traceId, setup);
-      if (transaction) {
-        return transaction;
+      const transactionwithErrorCount = await getTransactionWithErrorCount(
```

Comment:
On line 22 in this file:
```js
traceId: Joi.string().allow('') // TODO: this should be a path param and made required by 7.0
```

Can you make `traceId` required on the API level as well?

Reply:
We still want to make it a path param, but that can wait.